man/man5/vdev_id.conf.5

```diff
@@ -169,6 +182,27 @@ definitions - one per physical path.
 	channel 86:00.0  0         B
 .fi
 .P
+A configuration with enclosure_symlinks enabled.
+.P
+.nf
+	multipath yes
+	enclosure_symlinks yes
+
+	#          PCI_ID      HBA PORT     CHANNEL NAME
+	channel    05:00.0     1            U
+	channel    05:00.0     0            L
+	channel    06:00.0     1            U
+	channel    06:00.0     0            L
+.fi
+In addition to the disks symlinks, this configuration will create:
+.P
+.nf
+	/dev/by-enclosure/enc-L0
+	/dev/by-enclosure/enc-L1
+	/dev/by-enclosure/enc-U0
+	/dev/by-enclosure/enc-U1
+.fi
+.P
```

Comment:
Please update the `vdev_id.conf.sas_direct.example` file so it includes `enclosure_symlinks yes`.

Reply:
Will this be limited to sas_direct? It seems as though it could apply to multipath as well

Reply:
@gdevenyi yep it will work (I tested with multipath).  multipath is independent of the HBA topology mapping (sas_direct/sas_switch).